package.json

```diff
@@ -49,28 +49,64 @@
   },
   "scripts": {
     "bootstrap:hoist": "lerna bootstrap --hoist --force-local",
-    "clean": "lerna clean --yes && rm -rf node_modules || true",
     "clean:install": "rm package-lock.json; npm run clean && npm install",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p --config webpack.dev-site.config.js",
     "compile": "lerna run compile --parallel",
     "danger": "danger ci",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
+    "heroku-prebuild": "npx lerna init",
     "jest": "jest",
-    "mocha": "cd packages/terra-functional-testing && npm run mocha",
-    "lint": "npm run lint:js",
     "lint:js": "eslint --ext .js,.jsx .",
+    "lint": "npm run lint:js",
+    "mocha": "cd packages/terra-functional-testing && npm run mocha",
     "pack": "webpack -p",
     "postinstall": "link-parent-bin && npm run bootstrap:hoist",
+    "start": "webpack-dev-server --config webpack.dev-site.config.js",
+    "start-heroku": "node ./scripts/hack-express-server.js --port $PORT --site './build' ",
     "test": "npm run lint && npm run jest && npm run mocha && npm run wdio",
-    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack",
+    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js",
     "wdio-site": "SITE=build node ./packages/terra-functional-testing/bin/test-runner.js",
     "wdio-webpack": "node ./packages/terra-functional-testing/bin/test-runner.js",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack"
   },
   "devDependencies": {
+    "@babel/cli": "^7.10.5",
```

Comment:
i moved all dev dependencies from the packages to here.

Reply:
When running jest in the scoped package, are there any issues?